.goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+builds:
+  - <<: &build_defaults
+      id: atmo
+      main: .
+      binary: atmo
+      env:
+        - CGO_ENABLED=1
+      goos:
+        - linux
+        - darwin
+      goarch:
+        - amd64
+        - arm64
+      ldflags:
+        - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
+      overrides:
+        - goos: linux
+          ldflags:
+            - -extldflags "-static"
+  - <<: *build_defaults
+    id: atmo-proxy
+    binary: atmo-proxy
+    tags: proxy
+
+changelog:
+  skip: true
```

Comment:
any reason not to introduce a changelog generator?

Reply:
We don't use conventional commits everywhere yet

Reply:
We generate our own AFAIK